src/mixins/widget.js

```diff
@@ -50,19 +57,28 @@ Read more on using connectors: https://alg.li/vue-custom`
       this.widget.dispose &&
       this.instantSearchInstance.started // a widget can't be removed if IS is not started
     ) {
-      this.instantSearchInstance.removeWidget(this.widget);
+      this.root.removeWidgets([this.widget]);
     }
   },
+  computed: {
+    root() {
```

Comment:
This is not the root but rather the parent index, right?

Reply:
yes, can call it like that as well, I meant root as in nearest node, but that's indeed wrong

Reply:
I also find `root` misleading. `parent` or `parentIndex` would be clearer?